ci-operator/step-registry/assisted/common/setup/test/assisted-common-setup-test-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: assisted-common-setup-test
+  from: assisted-test-infra-dev-script
+  commands: assisted-common-setup-test-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 500Mi
+  credentials:
+  - namespace: test-credentials
+    name: vsphere-template-ssh-keys
+    mount_path: /var/run/vault/sshkeys
+  - namespace: test-credentials
+    name: edge-group-assisted-installer-bot
+    mount_path: /var/run/assisted-installer-bot
+  timeout: 180m0s
+  documentation: |-
+    Checkout and execute assisted-test-infra on a remote server
```

Comment:
hmm, it cannot be the same documentation for all steps
what does it actually do?

Reply:
same thing for other steps. we need some explanation to do because setup-configure, setup-test, setup-prepare doesn't tell much

Reply:
seems unhandled